dds/idl/v8_generator.cpp

```diff
@@ -206,6 +414,32 @@ bool v8_generator::gen_struct(AST_Structure*, UTL_ScopedName* name,
       "    return OpenDDS::DCPS::copyToV8(*static_cast<const " << lname
                << "*>(source));\n"
       "  }\n\n"
+      "  void* fromV8(const v8::Local<v8::Object>& source) const\n"
+      "  {\n"
+      "    " << lname << "* result = new " << lname << "();\n"
```

Comment:
The heap allocation here doesn't seem to have a corresponding deallocation.  It makes sense to allocate here since we need to have a `void*` interface.  I think we need an additional virtual method to handle deallocation, to be called by node-opendds.cpp.  See MetaStructImpl (metaclass_generator.cpp) for another example where we need to manipulate data samples indirectly via a `void*`.

Reply:
Good catch, I'll add a deletion method and update the node-opendds PR to use it.